.github/files/homebrew.martin.rb.j2

```diff
@@ -23,11 +23,11 @@ class Martin < Formula
   on_linux do
     on_arm do
       sha256 "{{ linux_arm_sha256 }}"
-      url "https://github.com/maplibre/martin/releases/download/v#{current_version}/martin-aarch64-unknown-linux-musl.tar.gz"
+      url "https://github.com/maplibre/martin/releases/download/v#{current_version}/martin-aarch64-unknown-linux-gnu.tar.gz"
```

Comment:
This opens us up to this breaking due to different libc versions.

I'd like to avoid this, unless you have benchmarked the performance and for you, this makes a difference.

Open to alternative opinions ^^

Reply:
Okay I reverted this change, it doesn't affect me. From my read it seemed like using musl is unexpected by Homebrew on Linux but if it's working IRL then no need to change it.